src/library/crate/cratetablemodel.h

```diff
@@ -29,6 +30,38 @@ class CrateTableModel : public BaseSqlTableModel {
     int addTracks(const QModelIndex& index, const QList<QString>& locations) final;
     CapabilitiesFlags getCapabilities() const final;
 
+    QString getModelSetting(QString name) override {
```

Comment:
- Move all implementations from .h into .cpp file
- Fix indentation

Reply:
I think the indentation is ok?  clang-format doesn't want to make any changes